Rename computeIntrinsicRatioInformation to computeIntrinsicSizeAndPreferredAspectRatio.

sammygill opened the pull request:
#### 31c11245b0efecb8a3b65ae6fb2669accddfc419
<pre>
Rename computeIntrinsicRatioInformation to computeIntrinsicSizeAndPreferredAspectRatio.
<a href="https://bugs.webkit.org/show_bug.cgi?id=296284">https://bugs.webkit.org/show_bug.cgi?id=296284</a>
<a href="https://rdar.apple.com/156333988">rdar://156333988</a>

Reviewed by Brandon Stewart and Brent Fulgham.

computeIntrinsicRatioInformation is a pretty vague name so let&apos;s rename
it to be a bit more descriptive. Since this function seems to have two
primary goals we can rename it computeIntrinsicSizeAndPreferredAspectRatio.
Although this is a bit more wordy, this does a better job of capturing
what this function does.

Rename the intrinsicRatio argument also to preferredAspectRatio since
that is what these functions are actually computing by checking the
aspect-ratio property and not just returning the ratio associated with
the replaced content.

Canonical link: <a href="https://commits.webkit.org/297705@main">https://commits.webkit.org/297705@main</a>
</pre>
<!--EWS-Status-Bubble-Start-->
https://github.com/WebKit/WebKit/commit/4473128d8b91e20fc099865c3ee5b1fcb8ad5b2d

| Misc | iOS, visionOS, tvOS & watchOS  | macOS  | Linux |  Windows |
| ----- | ---------------------- | ------- |  ----- |  --------- |
| [✅ 🧪 style](https://ews-build.webkit.org/#/builders/38/builds/112587 "Passed style check") | [✅ 🛠 ios](https://ews-build.webkit.org/#/builders/131/builds/32319 "Built successfully") | [✅ 🛠 mac](https://ews-build.webkit.org/#/builders/138/builds/22797 "Built successfully") | [✅ 🛠 wpe](https://ews-build.webkit.org/#/builders/5/builds/118786 "Built successfully") | [✅ 🛠 win](https://ews-build.webkit.org/#/builders/59/builds/63050 "Built successfully") 
| [✅ 🧪 bindings](https://ews-build.webkit.org/#/builders/9/builds/114549 "Passed tests") | [✅ 🛠 ios-sim](https://ews-build.webkit.org/#/builders/130/builds/32971 "Built successfully") | [✅ 🛠 mac-AS-debug](https://ews-build.webkit.org/#/builders/123/builds/40882 "Built successfully") | [✅ 🧪 wpe-wk2](https://ews-build.webkit.org/#/builders/34/builds/85689 "Passed tests") | [❌ 🧪 win-tests](https://ews-build.webkit.org/#/builders/60/builds/36310 "") 
| [✅ 🧪 webkitperl](https://ews-build.webkit.org/#/builders/11/builds/115534 "Passed tests") | [✅ 🧪 ios-wk2](https://ews-build.webkit.org/#/builders/132/builds/26312 "Passed tests") | [✅ 🧪 api-mac](https://ews-build.webkit.org/#/builders/18/builds/101281 "Passed tests") | [❌ 🧪 api-wpe](https://ews-build.webkit.org/#/builders/41/builds/65993 "Found 141 new API test failures: /WPE/TestLoaderClient:/webkit/WebKitWebView/loading-status, /WPE/TestWebKitWebView:/webkit/WebKitWebView/notification, /WPE/TestWebKitFindController:/webkit/WebKitFindController/options, /WPE/TestLoaderClient:/webkit/WebKitWebView/reload, /WPE/TestWebKitUserContentManager:/webkit/WebKitUserContentManager/script-message-received, /WPE/TestWebKitWebView:/webkit/WebKitWebView/run-javascript, /WPE/TestWebKitWebView:/webkit/WebKitWebView/disable-web-security, /WPE/TestLoaderClient:/webkit/WebKitWebView/load-bytes, /WPE/TestWebKitWebView:/webkit/WebKitWebView/notification-initial-permission-allowed, /WPE/TestWebKitWebContext:/webkit/WebKitWebContext/uri-scheme ... (failure)") | 
| | [✅ 🧪 ios-wk2-wpt](https://ews-build.webkit.org/#/builders/133/builds/25609 "Passed tests") | [✅ 🧪 mac-wk1](https://ews-build.webkit.org/#/builders/135/builds/19416 "Passed tests") | [✅ 🛠 wpe-cairo](https://ews-build.webkit.org/#/builders/65/builds/62545 "Built successfully") | 
| | [✅ 🧪 api-ios](https://ews-build.webkit.org/#/builders/13/builds/95702 "Passed tests") | [✅ 🧪 mac-wk2](https://ews-build.webkit.org/#/builders/136/builds/19491 "Passed tests") | [✅ 🛠 gtk](https://ews-build.webkit.org/#/builders/2/builds/122007 "Built successfully") | 
| | [✅ 🛠 vision](https://ews-build.webkit.org/#/builders/128/builds/39661 "Built successfully") | [✅ 🧪 mac-AS-debug-wk2](https://ews-build.webkit.org/#/builders/122/builds/29546 "Passed tests") | [✅ 🧪 gtk-wk2](https://ews-build.webkit.org/#/builders/1/builds/94556 "Passed tests") | 
| [![loading](https://user-images.githubusercontent.com/3098702/171232313-daa606f1-8fd6-4b0f-a20b-2cb93c43d19b.png) 🧪 services](https://ews-build.webkit.org/#/builders/28/builds/112082 "Build is in progress. Recent messages:") | [✅ 🛠 vision-sim](https://ews-build.webkit.org/#/builders/121/builds/40044 "Built successfully") | [✅ 🧪 mac-wk2-stress](https://ews-build.webkit.org/#/builders/8/builds/97517 "Passed tests") | [✅ 🧪 api-gtk](https://ews-build.webkit.org/#/builders/21/builds/94293 "Passed tests") | 
| [✅ 🛠 🧪 merge](https://ews-build.webkit.org/#/builders/19/builds/24077 "Built successfully and passed tests") | [✅ 🧪 vision-wk2](https://ews-build.webkit.org/#/builders/126/builds/39412 "Passed tests") | [✅ 🧪 mac-intel-wk2](https://ews-build.webkit.org/#/builders/137/builds/17211 "Passed tests") | [✅ 🛠 playstation](https://ews-build.webkit.org/#/builders/134/builds/35749 "Built successfully") | 
| | [✅ 🛠 tv](https://ews-build.webkit.org/#/builders/127/builds/39549 "Built successfully") | [✅ 🛠 mac-safer-cpp](https://ews-build.webkit.org/#/builders/120/builds/45037 "Built successfully") | | 
| | [✅ 🛠 tv-sim](https://ews-build.webkit.org/#/builders/125/builds/39186 "Built successfully") | | | 
| | [✅ 🛠 watch](https://ews-build.webkit.org/#/builders/129/builds/42521 "Built successfully") | | | 
| | [✅ 🛠 watch-sim](https://ews-build.webkit.org/#/builders/124/builds/40927 "Built successfully") | | | 
<!--EWS-Status-Bubble-End-->

stwrt commented on Source/WebCore/rendering/svg/legacy/LegacyRenderSVGRoot.h:
```diff
@@ -46,7 +46,7 @@ class LegacyRenderSVGRoot final : public RenderReplaced {
     bool isEmbeddedThroughFrameContainingSVGDocument() const;
 
-    void computeIntrinsicRatioInformation(FloatSize& intrinsicSize, FloatSize& intrinsicRatio) const override;
+    void computeIntrinsicSizeAndPreferredAspectRatio(FloatSize& intrinsicSize, FloatSize& preferredAspectRatio) const override;
```
For consistency with `RenderBox.h` I do not think we need the argument names.

```
virtual void computeIntrinsicSizeAndPreferredAspectRatio(FloatSize& /* intrinsicSize */, FloatSize& /* preferredAspectRatio */) const { }
```

sammygill replied:
Yeah I think it's a bit silly that we have this but I'm planning on moving this function over to RenderReplaced in a follow up so it should get resolved by that

stwrt reviewed (approved):
Overall this change looks good to me.

brentfulgham reviewed (approved):
LGTM, too!

sammygill committed:
Rename computeIntrinsicRatioInformation to computeIntrinsicSizeAndPreferredAspectRatio.
https://bugs.webkit.org/show_bug.cgi?id=296284
rdar://156333988

Reviewed by Brandon Stewart and Brent Fulgham.

computeIntrinsicRatioInformation is a pretty vague name so let's rename
it to be a bit more descriptive. Since this function seems to have two
primary goals we can rename it computeIntrinsicSizeAndPreferredAspectRatio.
Although this is a bit more wordy, this does a better job of capturing
what this function does.

Rename the intrinsicRatio argument also to preferredAspectRatio since
that is what these functions are actually computing by checking the
aspect-ratio property and not just returning the ratio associated with
the replaced content.

Canonical link: https://commits.webkit.org/297705@main
